test/test_api_server.py

```diff
@@ -0,0 +1,20 @@
+import requests
+
+
+# This test behaves randomly...
+# def test_search_simple():
+#     query = 'http://localhost:8085/api/search?q=emp'
+#
+#     response = requests.get(query).json()
+#     print response
+#
+#     assert len(response['docs']) == 10
+#     assert len(response['docs']) == response['hits']
```

Comment:
In my environment, this test behaves differently for each run. I ran the following commands:

```bash
$ docker run ...
$ pytest # the first time 
$ pytest # the second time
```

And the resutls were:
1. `assert len(response['docs']) == response['hits']` failed, `response['hits']` retunred 20
2. `assert len(response['docs']) == 10` failed, `len(response['docs'])` returned 2

Reply:
I figured out the cause of this randomness. It was because `test_installer.py` reinitializes Solr with a test configuration. #4 will force the test to run at the begging of unit-tests so we can be sure what's inside Solr. 

However, I'm still not sure why `assert len(response['docs']) == response['hits']` fails yet.  `response['hits']` still  retunres 20. So, there should be a bug.